packages/driver/cypress/integration/dom/visibility_spec.ts

```diff
@@ -289,6 +295,18 @@ describe('src/cypress/dom/visibility', () => {
 <div id="none" style='display: none;'>
   <span>parent display: none</span>
 </div>\
+`)
+
+      this.$parentPointerEventsNone = add(`\
+<div style="pointer-events: none;">
+  <span style="position: fixed; top: 20px;">parent pointer-events: none</span>
```

Comment:
Can't tell this HTML from the HTML below it on the page since it's the same content.

```suggestion
  <span style="position: fixed; top: 20px;">parent pointer-events: none; top: 20px</span>
```

Reply:
Should be updated now with the self-contained covering element.